runtime/vm/ObjectMonitor.cpp

```diff
@@ -196,10 +196,12 @@ objectMonitorEnterBlocking(J9VMThread *currentThread)
 			/* In a Concurrent GC where monitor object can *move* in a middle of GC cycle,
 			 * we need a proper barrier to get an up-to-date location of the monitor object */
 			j9objectmonitor_t volatile *lwEA = VM_ObjectMonitor::inlineGetLockAddress(currentThread, J9MONITORTABLE_OBJECT_LOAD(currentThread, &((J9ThreadMonitor*)monitor)->userData));
-			j9objectmonitor_t lockInLoop = *lwEA;
+			// TODO: does macro break volatile?
```

Comment:
```suggestion
			/* TODO: does macro break volatile? */
```

Reply:
Comment removed (I updated the macros to have the volatile).